### ISSUE TITLE
Implement BiasedMF SGD training logic

### ISSUE THREAD

**Issue by Goldmite** [Goldmite/coffeeRecommender#28]
- [x] Initialize Model Parameters:
  - [x] Create user and coffee biases
  - [x] userFactors and coffeeFactors
- [x] Implement the prediction function
- [x] The Training Process with Stochastic Gradient Descent
- [x] update biases and latent factors using the Learning Rate (γ) and Regularization (λ)

**Comment by Goldmite** [comment]
- [x] save model to db
- [x] access current model
- [x] serialize model to save as binary and deserialize back - implement with protobuf for performance.
